packages/commonwealth/client/scripts/views/modals/ManageCommunityStakeModal/utils.ts

```diff
@@ -19,17 +19,11 @@ export const convertEthToUsd = (
   return (eth * rate).toFixed(2);
 };
 
-export const buildEtherscanLink = (ethChainId: number, txHash: string) => {
-  switch (ethChainId) {
-    case 81457:
-      return `https://blastscan.io/tx/${txHash}`;
-    case 84532:
-      return `https://sepolia.basescan.org/tx/${txHash}`;
-    case 11155111:
-      return `https://sepolia.etherscan.io/tx/${txHash}`;
-    default:
-      return `https://basescan.org/tx/${txHash}`;
-  }
+export const buildEtherscanLink = (txHash: string) => {
```

Comment:
Can you fix the usages? You just need to remove the ethChainId in the caller functions in useTransactionHistory.ts and TransactionSucceeded.tsx

Reply:
Yeah, just noticed this when I pushed the merge - will fix

Reply:
I merged the two approaches to make chainNodeId an optional parameter so that tx history screens can get multiple block explorers for a users transactions but chain scoped usage can ignore and use current chain